src/app/helptext/task-calendar/replication.ts

```diff
@@ -0,0 +1,155 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T('Replication task name'),
+
+    direction_placeholder: T('Direction'),
+    direction_tooltip: T('Direct the flow of the ZFS snapshots. Choose between pushing\
+ snapshots to a remote host or pulling snapshots from a remote host.'),
+
+    transport_placeholder: T('Transport'),
+    transport_tooltip: T('Method for snapshot transfer:<ul>\
+ <li><i>SSH</i> is supported by most systems. It requires a previously created\
+ <a href="%%docurl%%/system.html%%webversion%%#ssh-connection" target="_blank">SSH connection</a>.</li>\
+ <li><i>SSH+NETCAT</i> uses SSH to establish a connection to the remote system, then uses\
+ an unencrypted data stream to improve data transfer speeds. This is only an option when\
+ replicating to a FreeBSD system that has <a href="https://github.com/freenas/py-libzfs"\
+ target="_blank">py-libzfs</a> installed.</li>\
+ <li><i>LOCAL</i> replicates snapshots to another dataset within the system.</li>\
+ <li><i>LEGACY</i> uses the replication engine implemented in FreeNAS 11.2 and earlier.</li></ul>'),
+
+    ssh_credentials_placeholder: T('SSH Connection'),
+    ssh_credentials_tooltip: T('Configured SSH connection to use for the replication.\
+ Choose from a list of connections configured in System > <a href="%%docurl%%/system.html%%webversion%%#ssh-connection"\
+ target="_blank">SSH connection</a>.'),
+
+    netcat_active_side_placeholder: T('Netcat Active Side'),
+    netcat_active_side_tooltip: T('System that will open ports for the direct connection'),
+
+    netcat_active_side_listen_address_placeholder: T('Netcat Active Side Listen Address'),
+    netcat_active_side_listen_address_tooltip: T(''),
+
+    netcat_active_side_port_min_placeholder: T('Netcat Active Side Min Port'),
+    netcat_active_side_port_min_tooltip: T('Lowest port number open to connections.'),
+
+    netcat_active_side_port_max_placeholder: T('Netcat Active Side Max Port'),
+    netcat_active_side_port_max_tooltip: T('Highest port number open to connections.'),
+
+    netcat_passive_side_connect_address_placeholder: T('Netcat Active Side Connect Address'),
+    netcat_passive_side_connect_address_tooltip: T(''),
+
+    source_datasets_placeholder: T('Source Datasets'),
+    source_datasets_tooltip: T('From the source computer with snapshots to replicate, choose an\
+ existing ZFS pool or dataset with an active periodic snapshot task. Multiple source datasets can be chosen.'),
+
+    target_dataset_placeholder: T('Target Datasets'),
+    target_dataset_tooltip: T('Enter the ZFS pool/dataset on the remote or destination system which will\
+ store snapshots. Example: Poolname/Datasetname, not the mountpoint or filesystem path'),
+
+    recursive_placeholder: T('Recursive'),
+    recursive_tooltip: T('Include all child dataset snapshots in the replication.'),
```

Comment:
Not clear.

Reply:
Probably should be something like "replicate child datasets too"